### PR TITLE
Issue #214 mozhttpd download

### PR DESCRIPTION
**Pull request by jasonwist**
Hi,

I tried the git commands you suggested. I think they worked.

Here is what I got to before I had posted a few days ago.

Is this the best way to show you what I have been working on/what I am having troubles with?

### CODE REVIEW

File: tests/direct_scraper/test_direct_scraper.py [hunk 1]
```diff
@@ -23,7 +26,7 @@ def tearDown(self):
         mozfile.rmtree(self.temp_dir)
 
     def test_url_download(self):
-        test_url = 'https://mozqa.com/index.html'
+        test_url = urljoin(self.wdir, 'download_test.txt')
```
Comment (whimboo) [inline tests/direct_scraper/test_direct_scraper.py]: The problem why this is not working is that `setUp()` is not calling itself via the base class. But given that the code for creating the tmp folder is already present in `MozHttpdBaseTest.setUp` you can get rid of the whole method. The same applies also for `tearDown`.

File: .gitignore [hunk 2]
```diff
@@ -2,4 +2,4 @@
 *.pyc
 build
 dist
-tests/venv
+tests/venv
```
Comment (whimboo) [inline .gitignore]: nit: this change can be reverted.

**Comment by whimboo** [comment]
Btw. this will become the fix for issue #214.

**Comment by Nebelhom** [comment]
Hi @jasonwist ,

are you still working on this PR?

If you are stuck with it, just write a comment an we will see if we can help you out :)

**Comment by whimboo** [comment]
Going to close this commit given that we haven't seen a response in the last 3 months. Feel free to reopen if you think that you will continue. Thanks.

**Pull request closed without merging**